internal/store/root/fsck.go

```diff
@@ -10,14 +10,18 @@ import (
 )
 
 // Fsck checks all stores/entries matching the given prefix.
-func (s *Store) Fsck(ctx context.Context, path string) error {
+func (s *Store) Fsck(ctx context.Context, store, path string) error {
```

Comment:
You'll need to change the usage in our tests. LGTM otherwise

Reply:
Yes, I saw that. Thank you.

I'll fix that once I get a chance to.